lib/stytch/b2b_organizations.rb

```diff
@@ -170,13 +410,69 @@ def update(
         put_request("/v1/b2b/organizations/#{organization_id}/members/#{member_id}", request)
       end
 
+      # Deletes a Member specified by `organization_id` and `member_id`.
+      #
+      # == Parameters:
+      # organization_id::
+      #   Globally unique UUID that identifies a specific Organization. The `organization_id` is critical to perform operations on an Organization, so be sure to preserve this value.
+      #   The type of this field is `String`.
+      # member_id::
+      #   Globally unique UUID that identifies a specific Member. The `member_id` is critical to perform operations on a Member, so be sure to preserve this value.
+      #   The type of this field is `String`.
+      #
+      # == Returns:
+      # An object with the following fields:
+      # request_id::
+      #   Globally unique UUID that is returned with every API call. This value is important to log for debugging purposes; we may ask for this value to help identify a specific API call when helping you debug an issue.
+      #   The type of this field is `String`.
+      # member_id::
+      #   Globally unique UUID that identifies a specific Member.
+      #   The type of this field is `String`.
+      # status_code::
+      #   The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+      #   The type of this field is `Fixnum`.
       def delete(
         organization_id:,
         member_id:
       )
         delete_request("/v1/b2b/organizations/#{organization_id}/members/#{member_id}")
       end
 
+      # Search for Members within specified Organizations. An array with at least one `organization_id` is required. Submitting an empty `query` returns all Members within the specified Organizations.
```

Comment:
`returns all Members` we have a hard upper limit of 100

Reply:
FYI, this is pulled from our docs, so we should update it there, too.

Reply:
sweet – also I was wrong, the default is `100` and the upper bound is `1000`